src/notebook/index.css

```diff
@@ -37,67 +37,79 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 7px;
+  padding-top: 7px;
+  padding-right: 7px;
+  padding-bottom: 7px;
```

Comment:
One last niggle. The shorthand `padding: 7px` can be used to set everything to the same value. Thanks for being patient!